src/graphql/schema/uuid/comment/resolvers.ts

```diff
@@ -0,0 +1,13 @@
+import { UserPayload } from '../user'
+import { CommentResolvers } from './types'
+
+export const resolvers: CommentResolvers = {
+  Comment: {
+    createdAt(comment) {
+      return comment.date.toString()
```

Comment:
```suggestion
      return comment.date.toISOString()
```

This should be the right function...